packages/elements/src/pagination/index.ts

```diff
@@ -38,37 +48,240 @@ export class Pagination extends BasicElement {
   }
 
   /**
-   * A `CSSResultGroup` that will be used
-   * to style the host, slotted children
-   * and the internal template of the element.
-   * @return CSS template
+   * Current page internal current page value
    */
-  static get styles (): CSSResultGroup {
-    return css`
-      :host {
-        display: block;
-        --responsive-width: 450;
-      }
-    `;
+  private _value = '';
+
+  /**
+   * Get internal current page value
+   * @returns current page value
+   */
+  private get internalValue (): number {
+    let value = parseInt(this._value, 10) || 1;
```

Comment:
You check against `0` later on so just convert `NaN` to `0` here.
```suggestion
    let value = parseInt(this._value, 10) || 0;
```

Reply:
Fixed